### DISCUSSION TITLE
Can the digital signature be removed from the *.exe file when generating RCP app?

### DISCUSSION THREAD

**Discussion by Phillipus** [eclipse-tycho/tycho#353] [Q&A] [answered]
Hi,

when an RCP app is built using Tycho, the product creates a Windows `YourApp.exe` file that contains a digital signature but is not valid:

![signed](https://user-images.githubusercontent.com/600504/139675557-0b8f0f6c-3215-40ef-afca-cef22b2a95fb.png)

Is there some way to not have a signed exe file or to remove the signature as part of the build process?

**Comment by laeubi** [comment, marked as answer]
There are some tools but no straigh forward way to do so see

- https://stackoverflow.com/questions/341168/can-i-remove-a-digital-signature-from-a-dll
- https://stackoverflow.com/questions/63326750/remove-digital-signature-from-generated-rcps-exe

**Reply by Phillipus** [reply]
Thanks, I assumed that would be the case but just thought I'd check.

**Reply by laeubi** [reply]
There are some issues in the Bugzilla, so if you think an (unsigned) version would be usefull it would be best to ask there as tycho technically only consumes the exe but do not create and/or sign it.

**Reply by Phillipus** [reply]
For reference - https://bugs.eclipse.org/bugs/show_bug.cgi?id=565937
